docs/en/flusso_autenticazione.rst

```diff
@@ -22,16 +22,16 @@ In the following, the descriptions of the flow steps, with the numbers indicated
      
      * In the SPID case, choses the authentication OP.
 
-  #. The RP prepares an Authorization Request with the parameters needed by *PKCE* and sends it to the *Authorization Endpoint* of the OP.
+  #. The RP prepares an Authorization Request and sends it to the *Authorization Endpoint* of the OP.
 
-  #. The OP authenticates the user by the credentials input and obtains, from the RP, the permission to access the user's attributes.
+  #. The OP authenticates the user and received the user's consent to release his attributes to the RP.
 
   #. The OP redirects the user to the URL contained in the parameter *redirect_uri* specified by the RP, passing an *Authorization Code* in the Authorization Response.
 
   #. The RP sends the *Authorization Code* received at the OP *Token Endpoint*.
 
   #. The OP *Token Endpoint* releases an **ID Token**, an **Access Token** and, if expected, a **Refresh Token**.
 
-  #. The RP receives and validates the **Access Token** and the **ID Token**. For asking the attributes that the user has authorized at the point 3, it sends a request to the OP *UserInfo Endpoint* ans uses, for the authentication, the **Access Token** contained in the HTTP Authorization header.
+  #. The RP receives and validates the **Access Token** and the **ID Token**. Then requests the user's attributes to the OP *UserInfo Endpoint* ans uses, for the authentication, the **Access Token** contained in the HTTP Authorization header.
```

Comment:
typo: ans --> and

Reply:
https://github.com/italia/spid-cie-oidc-docs/pull/81/commits/c0a12c7add6d6b7c592f8e444fb770ed1667a0da